leptonai/cli/photon.py

```diff
@@ -541,7 +541,7 @@ def run(
                 )
                 sys.exit(1)
             photon.launch(port=port)
-        except ModuleNotFoundError as e:
```

Comment:
strange, how did it pass ruff lint before (unused variable)

Reply:
I don't know lol, but yeah, removed...